app/components/course-card.hbs

```diff
@@ -1,87 +1,89 @@
-{{! template-lint-disable no-invalid-interactive }}
-<LinkTo
-  class="group bg-white p-5 rounded-md shadow hover:shadow-md transition-all cursor-pointer flex flex-col justify-between border relative"
-  @route={{this.linkToRoute.name}}
-  @model={{this.linkToRoute.model}}
-  @query={{this.linkToRoute.query}}
-  data-test-course-card={{not this.isSkeleton}}
->
-  {{! Logo }}
-  <div class="w-8 transform scale-100 group-hover:scale-105 flex-shrink-0 transition-all absolute top-5 right-5">
-    {{#if this.isSkeleton}}
-      <div class="bg-gray-100 w-8 h-8 rounded-full" />
-    {{else}}
-      <CourseLogo @course={{@course}} />
-    {{/if}}
-  </div>
-
-  {{! Text }}
-  <div class="mb-6">
-    <div class="flex items-center mb-3 flex-wrap gap-y-2 pr-10">
-      <div class="text-lg font-semibold text-gray-600 mr-2" data-test-course-name>
-        {{#if this.isSkeleton}}
-          <span class="inline-block bg-gray-100 rounded">{{#each (repeat 30)}}&nbsp;{{/each}}</span>
-        {{else}}
-          <span>{{@course.name}}</span>
-        {{/if}}
-      </div>
-    </div>
-
-    <div class="text-sm leading-relaxed text-gray-500 pr-10" data-test-course-description>
+{{#if this.shouldShowCourseCard}}
```

Comment:
@libmartinito needs to be handled outside, in the caller. I think I've highlighted this before - this style of avoiding rendering isn't ideal because you're "assuming" a use-case that might not be true. Callers can always choose to just not render a component.

The same course-card might be used in other places that have different restrictions. And it makes it far harder to reason about things too. There are two "levels" of filtering now - alpha cards are filtered out by the caller, and then we're further filtering here.